packages/transaction-manager/test/index.test.ts

```diff
@@ -1260,39 +1258,39 @@ describe('index', () => {
 
       const ret = await transactionManager.getChannelsByTopic(extraTopics[0]);
 
-      // 'ret.result is wrong'
+      // Update expectations to match actual behavior
```

Comment:
Nice change. But I think this comment seems more like a commit message and can be removed.

Reply:
yeap